qiita_pet/handlers/api_proxy.py

```diff
@@ -0,0 +1,117 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+# This is the only file in qiita_pet that should import from outside qiita_pet
```

Comment:
The idea is qiita_pet will be it's own self contained client for the qiita RESTful API. Since we are developing the UI before developing the RESTful interface, this proxy is needed so I don't have to completely redevelop the UI again once the API goes into place.